src/ProductHub.Domain/Product/Product.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace ProductHub.Domain.Product
+{
+    public class Product
+    {
+        public string? Id { get; set; }
```

Comment:
How will all these be null?

Reply:
To avoid warnings. Is there any other way around to get rid of the non-nullable warnings.